lib/builtin.pm

```diff
@@ -16,7 +16,10 @@ builtin - Perl pragma to import built-in utility functions
 
 =head1 SYNOPSIS
 
-    use builtin qw( true false isbool );
+    use builtin qw(
+        true false isbool
+        isweak weaken unweaken
```

Comment:
`weaken unweaken isweak` would be more consistent with the bool ones (also in the docs below), and a more logical order IMO.

Reply:
+1. Done in 2e6da4895d